app/client/src/features/events/Questions/AskQuestion/ValidationError.tsx

```diff
@@ -0,0 +1,10 @@
+import { GraphQLError } from 'graphql';
+
+class ValidationError extends GraphQLError {
+    constructor(message: string) {
+        super(message);
+        this.message = message;
+    }
+}
+
+export default ValidationError;
```

Comment:
Hmm, I'm unclear on why this is being made.  I believe a GraphQLError is usually something that's internal to GraphQL, although I could be wrong.  Do you mind explaining the motivation behind making a new Error class and extending GraphQLError?

Reply:
the main reason why I decided to create that class was because when I tried to just throw a string in AskQuestion.tsx, it gave me an error saying I could only throw an object of a class for some reason. I'm not sure if there's a better way around that

Reply:
Hmm, when you say throw a string do you mean `throw new 'My Error Message'`?  The normal way to throw an error would be to use the built in one `throw new Error('My Error Message')`. [ Here's some docs on error classes available in the browser. ](https://developer.mozilla.org/en-US/docs/web/javascript/reference/global_objects/error)